roles/general/tasks/swapfile.yml

```diff
@@ -16,6 +16,12 @@
 - name: Put the swap entry into fstab
   mount: name=none src={{ swap_file.path }} fstype=swap opts=sw passno=0 dump=0 state=present
 
+- name: Check swapon swapfile
+  raw: swapon | grep "{{ swap_file.path }}"
```

Comment:
Is there a reason you're using `raw` instead of `shell` or `command`? [This page](http://docs.ansible.com/ansible/latest/raw_module.html) says:

> Executes a low-down and dirty SSH command, not going through the module subsystem. This is useful and should only be done in two cases. The first case is installing python-simplejson on older (Python 2.4 and before) hosts that need it as a dependency to run modules, since nearly all core modules require it. Another is speaking to any devices such as routers that do not have any Python installed. In any other case, using the shell or command module is much more appropriate.